gef-ui/templates/partials/amendments/facility-value.njk

```diff
@@ -0,0 +1,83 @@
+{% extends "index.njk" %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% block pageTitle -%}
+  New facility value
+{%- endblock %}
+
+{% set headingHtml %}
+  <header>
+    <span class="govuk-caption-l" data-cy="heading-caption">
+      {{ exporterName }}
+    </span>
+    <h1 class="govuk-label-wrapper">
+      <label class="govuk-label govuk-label--xl govuk-!-padding-bottom-3" for="facility-value">
+        New facility value
+      </label>
+    </h1>
+  </header>
+{% endset %}
+
+{% block content %}
+  {% if errors %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors.errorSummary,
+      attributes: {
+        'data-cy': 'error-summary'
+      },
+      classes: "govuk-!-margin-top-4 govuk-!-margin-bottom-4"
+    }) }}
+  {% endif %}
+
+  {{ govukBackLink({
+    text: "Back",
+    href: previousPage,
+    attributes: {
+      'data-cy': 'back-link'
+    }
+  }) }}
+
+  <div class="govuk-grid-row govuk-!-margin-bottom-3">
+    <div class="govuk-grid-column-three-quarters">
+      <form method="post">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+        {{ govukInput({
+          id: "facility-value",
+          classes: "govuk-input--width-10",
+          label: {
+            html: headingHtml,
+            isPageHeading: true
+          },
+          value: facilityValue,
+          prefix: {
+            text: currencySymbol,
+            attributes: {
+            'data-cy': 'facility-value-prefix'
+          }
+          },
+          spellcheck: false,
+          attributes: {
+            'data-cy': 'facility-value'
+          }
+        }) }}
+
+        <div class="govuk-button-group">
+          {{ govukButton({
+            text: "Continue",
+            attributes: {
+              'data-cy': 'continue-button'
+            }
+          }) }}
+
+          <a href= {{"/case/" + dealId + "/facilities/" + facilityId + "/amendment/" + amendmentId + "/cancel"}} data-cy="cancel-link" class="govuk-link">Cancel</a>
```

Comment:
It looks to be working for me

Reply:
Learnt something new about nunjucks! 

I am wondering if we should change it though to be consistent, there are no other instances I can see that have such href constructions.

Equally this is cleaner.

An alternative which i'd much prefer, but would be a new pattern (want to be consistent), would be to construct the URL in the controller so the nunjucks templates just consume a variable e.g `cancelUrl`.

What do you think?

Reply:
I've gone for the latter - we're already passing in `previousPage` because the logic is quite complex, I therefore dont think its too big a break in pattern to pass in the cancelUrl too

Reply:
Nice one @MarRobSoftwire , didn't realise 👍